lib/titlecaps.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Title Caps
+ *
+ * Ported to JavaScript By John Resig - http://ejohn.org/ - 21 May 2008
+ * Original by John Gruber - http://daringfireball.net/ - 10 May 2008
+ * License: http://www.opensource.org/licenses/mit-license.php
+ */
+
+
+var small = "(a|an|and|as|at|but|by|en|for|if|in|of|on|or|the|to|v[.]?|via|vs[.]?)";
+var punct = "([!\"#$%&'()*+,./:;<=>?@[\\\\\\]^_`{|}~-]*)";
+
+titleCaps = function(title){
+	var parts = [], split = /[:.;?!] |(?: |^)["Ò]/g, index = 0;
```

Comment:
Not a clue, this is just taken directly from Resig/Gruber's implementation... I prefered not to change it.